tests/Aspire.Dashboard.Tests/Integration/IntegrationTestHelpers.cs

```diff
@@ -78,6 +79,30 @@ public static class IntegrationTestHelpers
         return dashboardWebApplication;
     }
 
+    public static HttpClient CreateHttpClient(
+        string address,
+        Action<X509Certificate2?>? validationCallback = null,
+        X509CertificateCollection? clientCertificates = null)
+    {
+        var handler = new SocketsHttpHandler
```

Comment:
For the resource service client, we allow configuration of SSL options via `configuration.Bind("Dashboard:ResourceServiceClient:Ssl", httpHandler.SslOptions);`. Do we want something similar here?

Reply:
This is just test code. In the real world, the client in this situation is the OTEL SDK.